Multisig/Logic/Ethereum/BlockchainDomainManager.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  BlockchainDomainManager.swift
+//  Multisig
+//
+//  Created by Johnny Good on 1/20/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import UnstoppableDomainsResolution
+
+class BlockchainDomainManager {
+    
+    let ens: ENS;
+    let resolution: Resolution;
+    
+    init() {
+        ens = ENS(registryAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e");
+        self.resolution = try! Resolution();
```

Comment:
is this guaranteed not to throw?

Reply:
@DmitryBespalov, As long as we are using default settings this won't throw, this, however, does limit the app to the mainnet only. 
I can reconfigure the library to use testnet as well for future development. We support Rinkeby for .crypto. 

Where can I get network configurations such as chainId and blockchain provider (I assume it is going to be infura as in gnosis-android) to do so?

Reply:
Got it, yes, that would be good to have because we have versions of the app working on Rinkeby network (in the future also on other chains). 

To get the blockchain provider url: `App.configuration.services.ethereumServiceURL`

To get the network name (we don't have chainId specified) - `App.configuration.app.network`

Reply:
Awesome, I have connected the library to the potential testnets and watched the error in case if network supplied is not supported by UD.